packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -94,6 +94,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox,
    */
   onMenuDismissed?: () => void;
 
+  /**
+   * Function that gets invoked before the menu gets dismissed
+   */
+  onMenuDismiss?: () => void;

```

Comment:
Why is onMenuDismissed not enough?

Reply:
I have a component which is using the ComboBox which needs to perform an action when the menu is being dismissed (it's too late once the menu has *been* dismissed).